sdk/src/main/java/com/amplitude/experiment/util/FetchException.kt

```diff
@@ -0,0 +1,8 @@
+package com.amplitude.experiment.util
+
+import okio.IOException
+
+class FetchException internal constructor(
```

Comment:
Use `internal` scoping for this class, and generally classes that customers dont need to interact directly with. The goal of an SDK developer is to create the most minimal API possible. Only expose what the developer needs.